blocks/algorithms/__init__.py

```diff
@@ -245,22 +248,18 @@ def compute_steps(self, gradients):
 
         Returns
         -------
-        An ordered dictionary of the same form as `gradient`, with the
-        proposed steps as values.
+        A tuple (steps, updates). `steps` is ordered dictionary of the
```

Comment:
Just to correct/clarify a little bit: @dwf is right that functions, objects, etc. should be in backquotes according to the NumPy docs, but judging from the documentation, only when referring to in-library objects (I couldn't find any ``` `int` ``` or ``` `tuple ` ``` for example). In our case, we use ```:class:`.Brick` ``` in many of those cases though, so that it automatically links things. Backquotes should be used for referring to parameters though (which is something you can't reference in Sphinx).